framework/include/st_things/st_things_types.h

```diff
@@ -297,12 +297,13 @@ typedef struct _st_things_representation {
 	 bool(*set_object_array_value) (struct _st_things_representation *rep, const char *key, const struct _st_things_representation **array, size_t length);
 
 } st_things_representation_s;
+typedef _st_things_representation_s st_things_representation_s;
```

Comment:
please follow the below
struct  name : XXX_s
typedef name : XXX_t

Reply:
Change. Please review.